3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,134 @@
+const getFullname = (
+  firstname,
+  surname,
+  useFormalName = false,
+  areYouMale = true
+) => {
+  if (
+    (firstname !== "" && surname !== "" && firstname.trim() !== "",
+    surname.trim() !== "")
+  ) {
+    useFormalName && areYouMale
+      ? console.log("lord " + firstname + " " + surname)
+      : useFormalName && !areYouMale
+      ? console.log("lady " + firstname + " " + surname)
+      : console.log(firstname + " " + surname);
```

Comment:
This was more optional, but it's great that you used it! Just next time, use it more in the way of my example: https://github.com/HackYourFuture-CPH/class27-homework/pull/81#discussion_r1391226009.

If you start using too many ternary operators in succession, it can become hard to read. In such cases, it's much better to use an `﻿if else` condition (as you did before). The code should always be clear and easy to read so your teammates understand what you are trying to do. I can see that you follow this rule perfectly. 👏